docs/quickstart.rst

```diff
@@ -117,35 +135,52 @@ we can evaluate metrics for subgroups within the data as below:
     >>> from fairlearn.metrics import MetricFrame
     >>> from sklearn.metrics import accuracy_score
     >>> from sklearn.tree import DecisionTreeClassifier
+    >>> from sklearn.model_selection import train_test_split
+    >>>
+    >>> X_train, X_test, y_train, y_test, \
+    A_train, A_test = train_test_split(X_ohe, y, race, random_state=123)
     >>>
     >>> classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
-    >>> classifier.fit(X, y_true)
+    >>> classifier.fit(X_train, y_train)
     DecisionTreeClassifier(...)
-    >>> y_pred = classifier.predict(X)
-    >>> mf = MetricFrame(metrics=accuracy_score, y_true=y_true, y_pred=y_pred, sensitive_features=sex)
+    >>> y_pred = (classifier.predict_proba(X_test)[:,1] >= 0.1)
+    >>> mf = MetricFrame(metrics=accuracy_score, y_true=y_test, y_pred=y_pred, \
+    sensitive_features=A_test)
     >>> mf.overall
-    0.8443...
+    0.529185173538776
     >>> mf.by_group
-    sex
-    Female    0.9251...
-    Male      0.8042...
+    race
+    AfricanAmerican    0.542303
+    Asian              0.690323
+    Caucasian          0.520261
+    Hispanic           0.595528
+    Other              0.561798
+    Unknown            0.591525
     Name: accuracy_score, dtype: float64
 
-Additionally, Fairlearn has lots of other standard metrics built-in, such as
+Note that above, because the outcome variable exhibits label imbalance, we set 
+the threshold for triggering a positive prediction lower 
+(to a probability of 0.1 or greater).
+Fairlearn has many standard metrics built-in, such as
 selection rate, i.e., the percentage of the population which have '1' as
 their label:
 
 .. doctest:: quickstart
     :options:  +NORMALIZE_WHITESPACE
 
     >>> from fairlearn.metrics import selection_rate
-    >>> sr = MetricFrame(metrics=selection_rate, y_true=y_true, y_pred=y_pred, sensitive_features=sex)
+    >>> sr = MetricFrame(metrics=selection_rate, y_true=y_test, y_pred=y_pred, \
```

Comment:
Selection rate is probably not the best criterion here. IMO (and thereby up for discussion) FNR is crucial, or perhaps the entire confusion matrix. FNR captures people that would get readmitted but we don't predict them to be, so that's real (medical) harm... 

FPs are the ones that wouldn't have been readmitted but we kept them in the hospital longer than needed so it's a cost but no harm to the person's health presumably.

So I suggest going with FNR/TPR parity or equalized odds (which captures the whole confusion matrix) as fairness criterion for mitigation with balanced accuracy as objective.

Reply:
I updated the portion that calculates the selection rate to instead show the false negative rate. I also added a few sentences to provide context to the example. 

I also updated the mitigation section to have the objective of balanced accuracy with equalized odds as the fairness criterion (unless you would prefer to use error rate as the criterion). I do want to quickly note that in the community call we discussed using balanced accuracy as the evaluation criteria in the initial section of the quickstart guide and decided to use accuracy instead with a lower threshold on the prediction probabilities because [to my understanding] balanced accuracy does not differentiate results by subgroups. But let me know if you want me to mock up what unmitigated balanced accuracy would look like, and we can discuss further here.

Reply:
True. Balanced is balanced by 0/1